tests/types/test-date32.py

```diff
@@ -127,6 +127,23 @@ def test_from_numpy_with_nats(np):
 
 
 
+#-------------------------------------------------------------------------------
+# Convert to/from Jay
+#-------------------------------------------------------------------------------
+
+def test_save_to_jay(tempfile_jay):
+    d = datetime.date
+    src = [d(1, 1, 1), d(2001, 12, 13), d(2026, 5, 9), None, d(1956, 11, 11)]
+    DT = dt.Frame(src)
+    DT.to_jay(tempfile_jay)
+    del DT
+    DT2 = dt.fread(tempfile_jay)
+    assert DT2.shape == (5, 1)
+    assert DT2.types == [dt.Type.date32]
+    assert DT2.to_list()[0] == src
```

Comment:
Do we also want to calculate stats on the date column and save/read it to/from jay?

Reply:
ok